device/cuda/src/clusterization/clusters_sum.cu

```diff
@@ -0,0 +1,81 @@
+
+#include <assert.h>
+
+#include "clusters_sum.hpp"
+#include "traccc/cuda/utils/definitions.hpp"
+
+#define CudaAssert(X)                                              \
```

Comment:
Doesn't look necessary either

Reply:
I used the definitions header to use the CUDA_ERROR_CHECK macro similar to what was done in the seeding kernels.
As you said the macros i defined are no longer used and i am removing them